components/paging/pager-load-more.js

```diff
@@ -129,20 +130,33 @@ class LoadMore extends PageableSubscriberMixin(FocusMixin(LocalizeCoreElement(Rt
 
 		const item = pageable._getItemByIndex(lastItemIndex + 1);
 
-		if (!item) return;
-		if (item.updateComplete) await item.updateComplete;
-
-		if (item.focus !== nativeFocus) {
-			requestAnimationFrame(() => item.focus());
-		} else {
-			const firstFocusable = getFirstFocusableDescendant(item);
-			if (firstFocusable) {
-				firstFocusable.focus();
-			} else if (item.focus === nativeFocus) {
-				item.tabIndex = -1;
-				requestAnimationFrame(() => item.focus());
+		let itemToFocus;
+		if (item) {
+			if (item.updateComplete) await item.updateComplete;
+			if (item.focus !== nativeFocus) {
+				itemToFocus = item;
+			} else {
+				const firstFocusable = getFirstFocusableDescendant(item);
+				if (firstFocusable) {
+					itemToFocus = firstFocusable;
+				} else if (item.focus === nativeFocus) {
+					item.tabIndex = -1;
+					itemToFocus = item;
+				}
 			}
 		}
+
+		if (itemToFocus) {
+			await new Promise(requestAnimationFrame);
+			itemToFocus.focus();
+		}
+
+		await new Promise(setTimeout);
+		this.dispatchEvent(new CustomEvent('d2l-pager-load-more-loaded', {
+			bubbles: false,
```

Comment:
Nit: these [are false by default](https://developer.mozilla.org/en-US/docs/Web/API/Event/Event#options), so if inclined, this statement could become:
```javascript
this.dispatchEvent(new CustomEvent('d2l-pager-load-more-loaded'));
```